array-manipulation-1/src/tail.js

```diff
@@ -0,0 +1,18 @@
+/* exported tail */
+
+function tail(array) {
+  var tailEnd = [];
+  for (var i = 0; i < array.length - 1; i++) {
+    tailEnd.push(array[i + 1]);
```

Comment:
Same deal here, if you start at `1` and end at the last index, you can just use `i` in this loop block.